nominees/notesmaster.json

```diff
@@ -0,0 +1,58 @@
+    },
+    {
+      "SDGNumber": 5,
+      "evidenceText": "In the OER content programs NotesMaster manages for the Commonwealth of Learning we use the Common Wealth of Learning Gender Mainstreaming Toolkit for Teachers and Teacher Educators as well as the Checklist for the Development of Gender-Responsive Learning Materials to ensure that gender equality is met in all subjects developed."
+    },
+    {
+      "SDGNumber": 9,
+      "evidenceText": "NotesMaster is an online education innovation in that it allows content to be created by local teachers, that is based around the local curriculum. The NotesMaster Library allows the content from all 31 countries to be easily accessed, used or remixed to suit local customs. The library also supports access in other popular learning platforms such as Edmodo, and Google Classroom."
+    },
+    {
+      "SDGNumber": 10,
+      "evidenceText": "Through free and open access to quality education resources for formal learners and also second chance learners. Notesmaster collaborated with its partner the Caribbean Examinations Council to implement the '2nd Chance Program' to the ‘at risk’ youth population (16-30 age group) in 9 regional territories of the Caribbean."
+    },
+    {
+      "SDGNumber": 11,
+      "evidenceText": "Through its partnership with the Namibian College of Open Learning, NotesMaster was pleased to be awarded a WSA for best societal impact for its digital resources. This program led to improved outcomes for communities in Namibia and for second chance learners aged 14-21."
+    },
+    {
+      "SDGNumber": 17,
+      "evidenceText": "Through a collaboration with Commonwealth of Learning, Ministry of Education Zambia, and local teachers, education officers and other partners, NotesMaster successfully piloted eLearning technologies in Muchinga Province, Zambia which in its first year saw an 11% increase (37% to 49%) in pass rates for secondary level exams. Muchinga province moved from 11th (last) on the results charts to 7th. The program is being scaled up in 2021-2022. The link shows the news coverage of the program, with the Provincial Education Officer, praising the achievements."
+    }
+  ],
+  "sectors": [
+    "Education"
+  ],
+  "type": [
+    "content"
```

Comment:
@notesmaster - could you provide links to the NotesMaster content on the new site so that we can verify the licensing?

Reply:
@conradsp 
I thought i'd make you a video to show the content and its license.
Video link: https://screencast-o-matic.com/watch/cr13XvV1rER

Here is a login to look at the lessons from the program in Trinidad and Tobago, scroll to the bottom to see the license:

Website: https://notesmaster.com/
Email: moett@nm.com
Password: Winthorpe76!

Physical Education lesson: https://notesmaster.com/classes/standard/9559/note/39585
Primary English lesson:  https://notesmaster.com/classes/standard/9536/note/115372